vision_evaluation/evaluators.py

```diff
@@ -816,3 +817,168 @@ def __init__(self):
         super().__init__(metric='SPICE')
         self.predictions = []
         self.targets = []
+
+
+class MattingEvaluatorBase(Evaluator):
+    """
+    Base class for image matting evaluator
+    """
+    def __init__(self):
```

Comment:
better to make the child class pass in the metric name from __init__

Reply:
good idea! has already changed!